ansible/configs/sap-integration/default_vars.yml

```diff
@@ -0,0 +1,384 @@
+        description: "Only from bastion"
+        from_port: 1
+        to_port: 65535
+        protocol: tcp
+        group: HanaSG
+        rule_type: Ingress
+
+# -------------------------------------------------------------------
+# OpenStack Instances
+# -------------------------------------------------------------------
+
+# Bastion Configuration
+bastion_instance_type: "2c2g30d"
+#bastion_instance_image: rhel-server-7.7-update-2
+bastion_instance_image: rhel-8.1
+# Root Filesystem Size
+bastion_rootfs_size: 30
+
+# Instance Types to be passed to OpenShift Installer
+master_instance_type: 4c16g30d
+worker_instance_type: 16c30g100d
+
+# Instances to be provisioned in new project
+# Provide these as a list.
+# Each instance type can have any number of replicas deployed with the same
+# configuration.
+# Metadata in OpenStack is equivelent to tags in AWS
+# These instances will be created with Cinder persistent volumes
+instances:
+- name: "bastion-{{ guid }}"
+  count: 1
+  unique: true
+  public_dns: true
+  dns_loadbalancer: true
+  floating_ip: true
+  image_id: "{{ bastion_instance_image }}"
+  sofware_image_id: "{{ sap_software_image }}"
+  flavor:
+    osp: "{{ bastion_instance_type }}"
+  tags:
+    - key: "AnsibleGroup"
+      value: "bastions"
+    - key: "ostype"
+      value: "linux"
+    - key: "instance_filter"
+      value: "{{ env_type }}-{{ email }}"
+    - key: ansible_python_interpreter
+      value: /usr/libexec/platform-python
+  rootfs_size: "{{ bastion_rootfs_size }}"
+  network: ocp
+  softwarefs_size: "{{ software_size }}"
+  security_groups:
+    - BastionSG
+
+- name: "hana-{{ guid }}"
+  count: 1
+  public_dns: false
+  dns_loadbalancer: false
+  floating_ip: false
+  image_id: "{{ hana_instance_image }}"
+  flavor:
+    osp: "{{ hana_instance_type }}"
+  tags:
+    - key: "AnsibleGroup"
+      value: "hanas"
+    - key: "ostype"
+      value: "rhel"
+    - key: "instance_filter"
+      value: "{{ env_type }}-{{ email }}"
+    - key: ansible_python_interpreter
+      value: /usr/libexec/platform-python
+  rootfs_size: "50"
+  network: ocp
+  volumes:
+    - volume_name: "hana_pv"
+      volume_size: "{{ pv_size_hana }}"
+  security_groups:
+    - HanaSG
+
+- name: "s4hana-{{ guid }}"
+  count: 1
+  public_dns: true
+  dns_loadbalancer: true
+  floating_ip: true
+  image_id: "{{ s4hana_instance_image }}"
+  flavor:
+    osp: "{{ s4hana_instance_type }}"
+  tags:
+    - key: "AnsibleGroup"
+      value: "s4hanas"
+    - key: "ostype"
+      value: "rhel"
+    - key: "instance_filter"
+      value: "{{ env_type }}-{{ email }}"
+    - key: ansible_python_interpreter
+      value: /usr/libexec/platform-python
+  rootfs_size: "50"
+  network: ocp
+  volumes:
+    - volume_name: "s4hana_pv"
+      volume_size: "{{ pv_size_s4hana }}"
+  security_groups:
+    - HanaSG
+
+# Variable used to identify the request being made by a Red Hatter or Partner (external)
+rh_internal: false
+
+# SAP HANA and S/4HANA instances variables
```

Comment:
Just for organization, can you move these up above the instance list with the others? Makes it a bit more streamline to read.

Reply:
done